workflow-templates/zephyr/README.md

```diff
@@ -0,0 +1,180 @@
+# Zephyr
+
+## Description
+
+Workflows in this group extend the workflows in [Basic](../basic/README.md)
+group and introduce several new functionalities which aid in the development of
+the projects using Zephyr.
+
+Specifically, they provide:
+
+- Everything that [Basic](../basic/README.md) workflows already do: changelog
+  preparation, tagging, publishing GitHub releases, etc.,
+- Running _builds_ on Pull Requests and during releases processes,
+- Caching of West modules and toolchains downloaded by East to speed up the
+  _build_ process,
+- A way to change what _build_ means for each project,
+- A way to specify which artefacts are attached to the published GitHub releases
```

Comment:
You are using artefacts and artifacts throughout this document, stick to one for consistency